linux_os/guide/services/base/service_messagebus_disabled/rule.yml

```diff
@@ -17,14 +17,15 @@ rationale: |-
     impractical for any system which needs to provide
     a graphical login session.
 
-severity: unknown
+severity: low 
 
 identifiers:
     cce@rhel6: 26913-4
     cce@rhel7: 80260-3
+    cce@rhel8: 82391-4
```

Comment:
@shawndwells I don't think this service is available on rhel8. If it no longer is, please return the CCE back to the pool.

Reply:
Done